docs_src/microservices/application/services/AvailableAppServices.md

```diff
@@ -2,8 +2,8 @@
 
 The following table lists the available EdgeX Application Services:
 
-| Repository                               | Status | Comments                                                                                                                                                                       | Documentation                                                |
-| ------------------------------------------------------------ | ------ |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------------------------ |
-| [app-service-configurable](https://github.com/edgexfoundry/app-service-configurable/tree/{{edgexversion}}) | Active | App Service which provides configurable function pipelines capability for built-in pipeline functions                                                                          | [app-service-configurable docs](./AppServiceConfigurable.md) |
-| [app-rfid-llrp-inventory](https://github.com/edgexfoundry/app-rfid-llrp-inventory/tree/{{edgexversion}}) | Active | App Service which generate Inventory movement Events from raw LLRP events produced by [device-rfid-llrp](https://github.com/edgexfoundry/device-rfid-llrp-go/tree/{{edgexversion}}) | [app-rfid-llrp-inventory docs](./AppLLRPInventory.md)        |
-| [app-record-replay](https://github.com/edgexfoundry/app-record-replay/tree/{{edgexversion}}) | Active | App Service for Development/Testing with capability to Record and Replay EdgeX Events                                                                                          | [app-record-replay docs](./AppRecordReplay.md)               |
+| Repository                               | Status | Comments                                                                                                                                                                       | Documentation                                                        |
```

Comment:
can we use the word Purpose or Description instead of Comments?

```suggestion
| Repository                               | Status | Purpose                                                                                                                                                                       | Documentation                                                        |
```

Reply:
Yep, I like it.

Reply:
done